### PR TITLE
volnoti: add module

### PR DESCRIPTION
**Pull request by IvanMalison**
### Checklist

- [x] Change is backwards compatible.

- [x] Code formatted with `./format`.

- [x] Code tested through `nix-shell --pure tests -A run.all`.

- [ ] Test cases updated/added. See [example](https://github.com/nix-community/home-manager/commit/f3fbb50b68df20da47f9b0def5607857fcc0d021#diff-b61a6d542f9036550ba9c401c80f00ef).

- [x] Commit messages are formatted like

    ```
    {component}: {description}

    {long description}
    ```

    See [CONTRIBUTING](https://github.com/nix-community/home-manager/blob/master/doc/contributing.adoc#sec-commit-style) for more information and [recent commit messages](https://github.com/nix-community/home-manager/commits/master) for examples.

- If this PR adds a new module

  - [x] Added myself as module maintainer. See [example](https://github.com/nix-community/home-manager/blob/068ff76a10e95820f886ac46957edcff4e44621d/modules/programs/lesspipe.nix#L6).

  - [ ] Added myself and the module files to `.github/CODEOWNERS`.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -339,3 +339,5 @@
 /modules/xresources.nix                               @rycee
 
 /modules/xsession.nix                                 @rycee
+
+/modules/services/volnoti.nix						  @IvanMalison
```
Comment (rycee) [inline .github/CODEOWNERS:343]: Should use spaces, not tabs.

**Comment by rycee** [comment]
Thanks for the contribution! Added a comment.

File: modules/services/volnoti.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  cfg = config.services.volnoti;
+
+in {
+  meta.maintainers = [ maintainers.imalison ];
+
+  options = {
+    services.volnoti = { enable = mkEnableOption "Volnoti volume HUD daemon"; };
+  };
+
+  config = mkIf cfg.enable {
```
Comment (berbiche) [inline modules/services/volnoti.nix:16]:
```suggestion
  config = mkIf cfg.enable {
    home.packages = [ pkgs.volnoti ];
```

To be able to use `volnoti-show` without the full path from the command line for instance.

File: modules/services/volnoti.nix [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    home.packages = [ pkgs.volnoti ];
```
Comment (berbiche) [inline modules/services/volnoti.nix:17]: I forgot to mention that we should make the package configurable.

Here's an example: https://github.com/nix-community/home-manager/blob/d2ade52d9616706f785f7b54956f814e2fd2ac9d/modules/programs/vivid.nix#L14-L19

**Review by berbiche — APPROVED** [review]
LGTM!

Thanks

**Comment by berbiche** [comment]
@IvanMalison hey, I didn't realize the package option was incorrectly nested.

Can you make a new PR that moves the package option under `services.volnoti`?

**Pull request merged**